browser/go/conversion-test/conversion_test.go

```diff
@@ -41,7 +41,7 @@ func TestBrowserConversions(t *testing.T) {
 
```

Comment:
`TestBrowserConversions` took 70 seconds for me the first time I ran it because I needed to download new packages from NPM. Can you change the context timeout to 120 seconds?

Reply:
Yep